### PR TITLE
[1LP][RFR] First integration test for VMware Cloud provider

### PR DESCRIPTION
**Pull request by miha-plesko**
With this commit we introduce VMwareCloudProvider class and implement it enough for following (exisiting) tests to pass:

- cfme/tests/cloud/test_providers.py::test_provider_add_with_bad_credentials
- cfme/tests/cloud/test_providers.py::test_provider_crud

Depends on: https://github.com/ManageIQ/wrapanapi/pull/209

### CODE REVIEW

File: cfme/common/provider.py [hunk 1]
```diff
@@ -1063,6 +1063,22 @@ def num_vm(self):
...
+    def num_availability_zone_ui(self):
+        return int(self.get_detail("Relationships", "Availability Zones"))
+
+    @variable(alias="db")
+    def num_orchestration_stack(self):
+        pass  # TODO: come up with a db query
+
+    @num_orchestration_stack.variant('ui')
+    def num_orchestration_stack_ui(self):
+        return int(self.get_detail("Relationships", "Orchestration Stacks"))
```
Comment (quarckster) [inline cfme/common/provider.py]: **QUESTION**
Are these properties common for all CloudInfraProviders subclasess? If not please put them into respectful class.

**Comment by miha-plesko** [comment]
@psav ping

**Comment by miha-plesko** [comment]
@psav I've rebased to latest master, tested both supported tests once again, and repushed. This should be ready to go IMAO. I kindly ask for a review.

NOTE: When running locally I had to manually edit `entry_points.txt` file since we're adding a new cloud provider. I think Travis CI is failing for this reason, because it doesn't get it updated.

**Comment by dajoRH** [comment]
Lint report for commit 2e0af835c12efc1f14b3319539b2668f9b88c7aa:

`cfme/tests/cloud/test_providers.py`:
- :red_circle: Line 17:1: **F401** *'cfme.cloud.provider.vcloud.VmwareCloudProvider' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by miha-plesko** [comment]
@psav I've rebased and repushed. Could you please take a look if we're good to go?

**Pull request merged**